DEVELOPMENT.md

```diff
@@ -1,3 +1,9 @@
+- Edit the `defaultValue` in `example/lib/main.dart` for the apiKey with the sample app to test:
```

Comment:
The env variable only works in CI, right? i.e.: if you have it in your `.bash_profile` or `.zshrc` it won't get picked up?

Reply:
so I am actually thinking about using `sed` like in the other repos and reemove this whole env variable thing

Reply:
yeah, that probably works best. I ended up doing it for iOS integration tests as well, since env variables are not exactly easy to work with on iOS